packages/components/src/Actions/ActionDropdown/ActionDropdown.test.js

```diff
@@ -110,79 +111,72 @@ describe('InjectDropdownMenuItem', () => {
 
 describe('Dropup', () => {
 	it('should switch to dropup when it is near the bottom of a tc-dropdown-container', () => {
-		// given
-		const classListContainingClass = {
-			contains() {
-				return true;
-			},
-		};
-		const classListNotContainingClass = {
-			contains() {
-				return false;
-			},
-		};
-		function getDropdownMenuElement(top, bottom) {
-			return {
-				classList: classListContainingClass,
-				getBoundingClientRect() {
-					return { top, bottom };
-				},
-			};
-		}
-		const divContainerElement = {
-			tagName: 'DIV',
-			classList: classListContainingClass,
-			getBoundingClientRect() {
-				return { top: 0, bottom: 35 };
-			},
-		};
-		const bodyElement = {
-			tagName: 'BODY',
-			classList: classListNotContainingClass,
-			getBoundingClientRect() {
-				return { top: 0, bottom: 35 };
-			},
-		};
-		function getActionDropdownEvent(top, bottom, container) {
-			return {
-				target: {
-					classList: classListContainingClass,
-					nextSibling: getDropdownMenuElement(top, bottom),
-					parentElement: {
-						classList: classListNotContainingClass,
-						// dropdown container
-						// simulate the tc-dropdown-container className
-						parentElement: container,
-					},
-				},
-			};
-		}
-		// dropdown that doesn't overflow
-		const event = getActionDropdownEvent(0, 25, divContainerElement);
-		// dropdown that is above the top of container
-		const topOverflowEvent = getActionDropdownEvent(-5, 10, divContainerElement);
-		// dropdown that is under the bottom of container
-		const bottomOverflowEvent = getActionDropdownEvent(20, 40, divContainerElement);
-		// dropdown that is under the bottom of body element
-		const bodyBottomOverflowEvent = getActionDropdownEvent(20, 40, bodyElement);
+		const isDropup = true;
+		const isDropdown = false;
 
-		const wrapper = shallow(<ActionDropdown items={[{ label: 'item 1' }, { label: 'item 2' }]} />);
-		expect(wrapper.state().dropup).toBeFalsy();
+		function testSwitch({
+			containerPosition,
+			menuPosition,
+			isInitialDropup,
+			isDropupOrIsDropdown,
```

Comment:
Either the naming or the data stored is weird. I don't understand what's the meaning. When seeing it in standalone, It seems to be always true, and when seeing with the usage, I just don't understand neither ^^'